spec/lib/metasploit/framework/login_scanner/jenkins_spec.rb

```diff
@@ -2,9 +2,103 @@
 require 'metasploit/framework/login_scanner/jenkins'
 
 RSpec.describe Metasploit::Framework::LoginScanner::Jenkins do
+  subject(:http_scanner) { described_class.new('host' => 'nosuchplace.metasploit.com') }
 
-    it_behaves_like 'Metasploit::Framework::LoginScanner::Base',  has_realm_key: true, has_default_realm: false
-    it_behaves_like 'Metasploit::Framework::LoginScanner::RexSocket'
-    it_behaves_like 'Metasploit::Framework::LoginScanner::HTTP'
```

Comment:
The login scanner should definitely continue to behave like a login scanner, where you running into issues with these tests?

Reply:
Yes because it now calls the `jenkins_uri_check` during initialization, requiring me to copy over the actual tests and stub the calls to make them pass.

It still behaves like an HTTP scanner, just has a different setup